pkg/ovirt/ovirtApiAccess.es6

```diff
@@ -0,0 +1,79 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import CONFIG from './config.es6';
+import { logDebug, logError } from '../machines/helpers.es6';
+
+export function ovirtApiGet (resource, custHeaders, failHandler) {
+    const headers = Object.assign({}, {
```

Comment:
Can we assume we work against 4.1.4 and up, so we can use pipelining and friends? (see http://www.ovirt.org/blog/2017/05/higher-performance-for-python-sdk/ )?